### ISSUE TITLE
SMF version is not exported in Dell S6100/Z9100 switches

### ISSUE THREAD

**Issue by paavaanan** [Azure/sonic-buildimage#2947]
**Description**
- For debugging purpose exported SMF version number for DellS6100/Z9100 platforms.
- sysfs varisble smf_version is not exported in Nov build.
- Below output is missing:
```
root@sonic:/home/admin/# cat /sys/devices/platform/SMF.512/hwmon/hwmon0/smf_version
16
```
- This happens only in November build. (master/march are not affected)

**Steps to reproduce the issue:**
1. Try sonic-broadcom.bin from November branch or sonic-broadcom.bin (build 59)
2. cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_version

**Describe the results you received:**
```
root@sonic-z9100-02:/sys/devices/platform/SMF.512/hwmon/hwmon1# cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_version
cat: /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_version: No such file or directory
```

**Describe the results you expected:**
```
root@sonic:/home/admin/# cat /sys/devices/platform/SMF.512/hwmon/hwmon0/smf_version
16
```
**Additional information you deem important (e.g. issue happens only occasionally):**

    **Output of `show version`:**
root@sonic-z9100-02:/sys/devices/platform/SMF.512/hwmon/hwmon1# show version
SONiC Software Version: SONiC.HEAD.59-f434b80
Distribution: Debian 9.9
Kernel: 4.9.0-8-amd64
Build commit: f434b80
Build date: Tue May 21 04:54:34 UTC 2019
Built by: johnar@jenkins-worker-7

Docker images:
REPOSITORY                 TAG                 IMAGE ID            SIZE
docker-syncd-brcm          HEAD.59-f434b80     109ca065975e        361MB
docker-syncd-brcm          latest              109ca065975e        361MB
docker-orchagent-brcm      HEAD.59-f434b80     457a28e1f396        286MB
docker-orchagent-brcm      latest              457a28e1f396        286MB
docker-lldp-sv2            HEAD.59-f434b80     e1a678a1d3fa        274MB
docker-lldp-sv2            latest              e1a678a1d3fa        274MB
docker-dhcp-relay          HEAD.59-f434b80     539590339b35        256MB
docker-dhcp-relay          latest              539590339b35        256MB
docker-database            HEAD.59-f434b80     dccf21e74819        255MB
docker-database            latest              dccf21e74819        255MB
docker-snmp-sv2            HEAD.59-f434b80     260c9e930002        289MB
docker-snmp-sv2            latest              260c9e930002        289MB
docker-teamd               HEAD.59-f434b80     cc5e7c3f797e        274MB
docker-teamd               latest              cc5e7c3f797e        274MB
docker-router-advertiser   HEAD.59-f434b80     63c3a4453dcd        254MB
docker-router-advertiser   latest              63c3a4453dcd        254MB
docker-platform-monitor    HEAD.59-f434b80     88b5a8d8fb72        286MB
docker-platform-monitor    latest              88b5a8d8fb72        286MB
docker-fpm-quagga          HEAD.59-f434b80     1ebe455a8774        281MB
docker-fpm-quagga          latest              1ebe455a8774        281MB

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by sridhar-ravindran** [comment]
Trying in 61 build
===========
S6100
=====
root@sonic:~# show version

SONiC Software Version: SONiC.HEAD.61-fbe55e9
Distribution: Debian 9.9
Kernel: 4.9.0-8-amd64
Build commit: fbe55e9
Build date: Fri May 31 11:39:21 UTC 2019
Built by: johnar@jenkins-worker-4



root@sonic:~# cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_firmware_ver
1.25
root@sonic:~# cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_version
16
root@sonic:~#



Z9100
=====

root@sonic:/home/admin# show version

SONiC Software Version: SONiC.HEAD.61-fbe55e9
Distribution: Debian 9.9
Kernel: 4.9.0-8-amd64
Build commit: fbe55e9
Build date: Fri May 31 11:39:21 UTC 2019
Built by: johnar@jenkins-worker-4

Platform: x86_64-dell_z9100_c2538-r0
HwSKU: Force10-Z9100-C32
ASIC: broadcom
Serial Number: CN0KY5C4CES0079U0034
Uptime: 07:02:38 up 13:17,  1 user,  load average: 0.15, 0.26, 0.17


 cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_firmware_ver
2.4
sonic:/home/admin# cat /sys/devices/platform/SMF.512/hwmon/hwmon1/smf_version
16
root@sonic:/home/admin#


This github issue can be closed

**Comment by sridhar-ravindran** [comment]
Hi,
Code was merged only with build number 60.  But this git issue was raised in 59. 
It is working fine for 61. 

https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/buildimage-brcm-201811/60/
Changes
1.	[devices]: Dell Hwmon S6100/Z9100 SFM version export (#2521) (detail / githubweb)
2.	[devices]: Export reboot_reason sysfs attribute for DellEMC S6100/Z9100 (detail / githubweb)
3.	[swss.sh] flush FDB table during cold start (#2933) (detail / githubweb)
4.	[devices]: Optics fixes in Dell Z9100/Z9264f platforms (#2936) (detail / githubweb)
5.	[monit] Set memory usage alert at 50% (#2939) (detail / githubweb)
6.	[logrotate] Enhance robustness (#2942) (detail / githubweb)
7.	[mellanox]: fix wrong type of paramerter (#2950) (detail / githubweb)
8.	[201811][utilities][swss] advance sub-module heads (#2953) (detail / githubweb)

Please close the github issue.
